### DISCUSSION TITLE
Skip SSL Verification for endpoint

### DISCUSSION THREAD

**Discussion by mabombo** [open-telemetry/opentelemetry-java-instrumentation#8680] [Q&A]
There is a way to skip ssl verification for an endpoint in Instrumentation crd or directly in java agent configuration?
I have an apm server with self sign certificate and I don't want mount ca cert on all pods.

**Comment by laurit** [comment]
As far as I know there isn't. Most of the agent configuration is handled by opentelemetry java sdk autoconfigure extension https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md You could send your traces to opentelemetry collector and configure the cert there.
